Dockerfile

```
@@ -74,7 +75,6 @@ RUN yarn compile
 # Copy the rest. (node_modules are excluded in .dockerignore)
 COPY . /app
 
-# Start json2capnp -> Relies on manually creating cache directory before
 # Start Node app
-CMD sh -c "cd services/json2capnp && pwd && ./json2capnp 2000 /app/examples/runtime/cache/demo_transition > /app/json2capnp.log &" && yarn build:prod && yarn start
+CMD yarn build:prod && yarn start
```

Hadolint (2.14.0)

[warning] Use arguments JSON notation for CMD and ENTRYPOINT arguments

(DL3025)